src/DotNetCore.CAP/Abstractions/CapPublisherBase.cs

```diff
@@ -67,6 +67,16 @@ public Task PublishAsync<T>(string name, T contentObj, IDbTransaction dbTransact
             return PublishWithTransAsync(name, contentObj, callbackName);
         }
 
+        public virtual void PublishWithMongoSession<T>(string name, T contentObj, object mongoSession = null, string callbackName = null)
```

Comment:
Rename `PublishWithMongoSession` to `PublishWithMongo`, because mongoSession is a optional parameter.